api/app.js

```diff
@@ -1,3 +1,5 @@
+import bodyParser from "body-parser";
+import cors from "cors";
```

Comment:
What was the reason for including these?

It's always good to comment your own PRs with explanations for things so it's clear to reviewers why things area happening